test/test_new_spec.lua

```diff
@@ -3,16 +3,26 @@ require('strict').on()
 
 local icu_date = require('icu-date')
 local tap = require('tap')
+local yaml = require('yaml')
 
 local test = tap.test("new")
-test:plan(2)
+test:plan(3)
 
 test:test("creates new instance", function(test)
               test:plan(1)
               local date = icu_date.new()
               test:isnt(date, nil)
 end)
 
+test:test("serialize", function(test)
+              test:plan(3)
+              local date = icu_date.new()
+              test:isnt(date, nil)
+              date:set_millis(1234567891011)
+              test:is(tostring(date), '2009-02-13T23:31:31.011Z')
+              test:like(yaml.encode(date), '2009%-02%-13T23%:31%:31%.011Z')
```

Comment:
Nit: Colon is not the [special character](https://www.lua.org/manual/5.3/manual.html#6.4.1).

Reply:
Fixed